packages/zowe-explorer/README.md

```diff
@@ -6,8 +6,6 @@
 [![slack](https://img.shields.io/badge/chat-on%20Slack-blue)](https://slack.openmainframeproject.org/)
```

Comment:
Should we add an OpenVSX badge to show that we have prereleases?
![](https://img.shields.io/open-vsx/v/zowe/vscode-extension-for-zowe)
<sup>https://img.shields.io/open-vsx/v/zowe/vscode-extension-for-zowe</sup>